.github/workflows/gh-pages.yml

```diff
@@ -46,6 +47,9 @@ jobs:
       - name: Build Sway reference
         run: mdbook build docs/reference
 
+      - name: Build Sway std library
+        run: forc doc --manifest-path ./sway-lib-std && cp sway-lib-std/out/doc/* docs/std
```

Comment:
```suggestion
        run: forc doc --manifest-path ./sway-lib-std && cp -r sway-lib-std/out/doc/* docs/std
```

Reply:
ah nice catch

Reply:
oh your below comment would basically negate this